src/selfplay/loop.cc

```diff
@@ -107,7 +110,7 @@ std::map<uint64_t, PolicySubNode> policy_subs;
 bool gaviotaEnabled = false;
 bool deblunderEnabled = false;
 float deblunderQBlunderThreshold = 2.0f;
-
+float deblunderQBlunderWidth = 0.0f;
```

Comment:
Are lines 112+113 actually necessary, or just confusing?

Reply:
The variables have to be declared - their values are meaningless since they are later set redundantly during initialization. But its fine.

Reply:
Thanks for the explanation!